DigitalLearningSolutions.Data.Tests/Services/RegistrationServiceTests.cs

```diff
@@ -840,111 +840,47 @@ public void RegisterDelegateByCentre_calls_data_service_to_set_prn()
-                () => registrationService.PromoteDelegateToAdmin(adminRoles, 1, 1)
-            );
-
-            // Then
-            result.Error.Should().Be(AdminCreationError.UnexpectedError);
-        }
-
         [Test]
         public void
-            PromoteDelegateToAdmin_throws_email_in_use_AdminCreationFailedException_if_active_admin_already_exists()
+            PromoteDelegateToAdmin_throws_AdminCreationFailedException_if_active_admin_already_exists()
         {
             // Given
-            var delegateUser = UserTestHelper.GetDefaultDelegateUser();
-            var adminUser = UserTestHelper.GetDefaultAdminUser();
+            const int userId = 2;
+            var adminAccount = UserTestHelper.GetDefaultAdminAccount(userId: userId);
             var adminRoles = new AdminRoles(true, true, true, true, true, true, true);
-            A.CallTo(() => userDataService.GetDelegateUserById(A<int>._)).Returns(delegateUser);
-            A.CallTo(() => userDataService.GetAdminUserByEmailAddress(A<string>._)).Returns(adminUser);
-
-            // When
-            var result = Assert.Throws<AdminCreationFailedException>(
-                () => registrationService.PromoteDelegateToAdmin(adminRoles, 1, 1)
-            );
 
-            // Then
-            using (new AssertionScope())
-            {
-                UpdateToExistingAdminAccountMustNotHaveHappened();
-                result.Error.Should().Be(AdminCreationError.EmailAlreadyInUse);
-            }
-        }
-
-        [Test]
-        public void
-            PromoteDelegateToAdmin_throws_email_in_use_AdminCreationFailedException_if_inactive_admin_at_different_centre_already_exists()
-        {
-            // Given
-            var delegateUser = UserTestHelper.GetDefaultDelegateUser();
-            var adminUser = UserTestHelper.GetDefaultAdminUser(centreId: 3, active: false);
-            var adminRoles = new AdminRoles(true, true, true, true, true, true, true);
-            A.CallTo(() => userDataService.GetDelegateUserById(A<int>._)).Returns(delegateUser);
-            A.CallTo(() => userDataService.GetAdminUserByEmailAddress(A<string>._)).Returns(adminUser);
+            A.CallTo(() => userDataService.GetAdminAccountsByUserId(userId)).Returns(new[] { adminAccount });
 
             // When
-            var result = Assert.Throws<AdminCreationFailedException>(
-                () => registrationService.PromoteDelegateToAdmin(adminRoles, 1, 1)
-            );
+            Action action = () => registrationService.PromoteDelegateToAdmin(adminRoles, 1, userId, 2);
 
             // Then
-            using (new AssertionScope())
-            {
-                UpdateToExistingAdminAccountMustNotHaveHappened();
-                result.Error.Should().Be(AdminCreationError.EmailAlreadyInUse);
-            }
+            action.Should().Throw<AdminCreationFailedException>();
+            UpdateToExistingAdminAccountMustNotHaveHappened();

```

Comment:
This assertion helping method can be cleaned up as we don't care about the setting password anymore. Also looks like the datamethod itself (SetPasswordByAdminId) is no longer used anywhere as it was originally set up for the pre-UAR admin promotion only.